README.md

```diff
@@ -1002,7 +1002,7 @@ Table of Contents
    * [wakatime.com](https://wakatime.com/) — Quantified self-metrics about your coding activity, using text editor plugins, limited plan for free.
 
 ## Analytics, Events and Statistics
-
+   * [Avo](https://avo.app/) — Simplified analytics release workflow. Single-source-of-truth tracking plan, type safe analytics tracking library, in-app debuggers, data observability to catch all data issues before you release. Free for 2 workspace members and 1 hour data observability lookback.
```

Comment:
Please keep the empty line between heading and list

thanks!

Reply:
Ahh of course! I've fixed it now ✨